pyvcloud/vcd/client.py

```diff
@@ -515,6 +514,8 @@ def __init__(self,
 
         self.fsencoding = sys.getfilesystemencoding()
 
+        self._is_loggedin_user_sys_admin = False
```

Comment:
variable seems too long. As we can expect this to be used extensively, can we shorten the name? how about is_sysadmin or anything better?